beprof/example.py

```diff
@@ -0,0 +1,15 @@
+import urllib.request
+from beprof.curve import Curve
+import numpy as np
+
+url = 'http://berkeleyearth.lbl.gov/auto/Global/Complete_TAVG_complete.txt'
```

Comment:
Does this dataset contain data sample which is rising and falling ? In the comment to #9 I said "see for example", not "use it".